grails-app/domain/org/pih/warehouse/inventory/CycleCount.groovy

```diff
@@ -0,0 +1,64 @@
+package org.pih.warehouse.inventory
+
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.core.User
+
+class CycleCount {
+
+    String id
+
+    Location facility
+
+    Date dateLastRefreshed
```

Comment:
now that I'm seeing this, I wonder if the dateLastRefreshed field would be better off in the CycleCountItem because that's where we're doing the actual QoH refresh. If we're in a recount and do a refresh, we only change QoH for the CycleCountItem with countIndex==1 (countIndex==0 stays unchanged). So it could be useful to have that date info at the item level to be able to clearly see why QoH changed between counts.

I don't know if you've had a chance to think about this yet. (For context, [this is the API that will use the field](https://pihemr.atlassian.net/wiki/spaces/OB/pages/3332505612/Cycle+Count+Items+API#Refresh-Cycle-Count-Quantity-On-Hand))

Reply:
tech huddle: this could be added to the cycle count item later